files/build-raw-sequencing-files.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+import numpy as np
+import requests
+from pathlib import Path
+from urllib.parse import urlparse
+
+STORAGE_FOLDER_PATH = "./files/unprocessed_files/"
+OUTPUT_PATH = "./files/out/raw_sequencing_data.tsv"
+DEEPCONSENSUS_URL = "https://raw.githubusercontent.com/human-pangenomics/HPRC_metadata/main/data/sample-files/hprc_metadata_sample_files_DEEPCONSENSUS.tsv"
+HIFI_URL = "https://raw.githubusercontent.com/human-pangenomics/HPRC_metadata/main/data/sample-files/hprc_metadata_sample_files_HiFi.tsv"
+ONT_URL = "https://raw.githubusercontent.com/human-pangenomics/HPRC_metadata/main/data/sample-files/hprc_metadata_sample_files_ONT.tsv"
+METADATA_URLS = [DEEPCONSENSUS_URL, HIFI_URL, ONT_URL]
+TABLE_URL = "https://raw.githubusercontent.com/human-pangenomics/HPRC_metadata/main/data/production/hprc-production-biosample-table.tsv"
```

Comment:
Can we call this "BIOSAMPLE_TABLE_URL" or similar to indicate the biosamples are in here?